app/models/user.rb

```diff
@@ -313,9 +313,7 @@ def cached_followed_tag_names
   end
 
   def suspended?
-    # TODO: [@jacobherrington] After all of our Forems have been successfully deployed,
```

Comment:
Verified via Blazer that the old "banned" role indeed no longer exists, so we can safely remove this code (which was meant to be extremely temporary).